starlark/src/eval/mod.rs

```diff
@@ -852,12 +851,12 @@ fn eval_block(block: &BlockCompiled, context: &mut EvaluationContext) -> EvalRes
     Ok(r)
 }
 
-fn eval_module(
+fn eval_module<T: FileLoader>(
     module: &Module,
     env: &mut Environment,
     type_values: &TypeValues,
     map: Arc<Mutex<CodeMap>>,
-    file_loader: Rc<dyn FileLoader>,
+    file_loader: &T,
```

Comment:
Can we change it to `&dyn FileLoader` as well instead of introducing a type parameter?